scripts/check-versions.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+set -e
+
+if [ -z "${VERSION}" ]; then
+  echo "VERSION variable must be set" >&2
+  exit 1
+fi
+
+if [ "${VERSION}" == "0.0.0-dev" ]; then
+  echo "dev build, skipping version checks"
+  exit 0
+fi
+
+ROOT_DIR="${0%/*}"
+# update PATH to prefer scripts relative to this one e.g. yq
+export PATH="${ROOT_DIR}:${PATH}"
+
+CHART_ROOT="${CHART_ROOT-$(readlink -f ${0%/*}/../chart)}"
+KUSTOMIZE_ROOT="${KUSTOMIZE_ROOT-$(readlink -f ${0%/*}/../config)}"
```

Comment:
Any reason we wouldn't use ROOT_DIR here?

```suggestion
CHART_ROOT="${CHART_ROOT-$(readlink -f ${ROOT_DIR}/../chart)}"
KUSTOMIZE_ROOT="${KUSTOMIZE_ROOT-$(readlink -f ${ROOT_DIR}/../config)}"
```